examples/tab_autocompletion.py

```diff
@@ -1,268 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
I'd recommend double-checking that you removed all references to these two deleted files.